ui/component/fileRenderFloating/view.jsx

```diff
@@ -374,97 +394,133 @@ export default function FileRenderFloating(props: Props) {
+              {isFloating && (
+                <Button
+                  title={__('Close')}
+                  onClick={() => {
+                    if (hasClaimInQueue) {
+                      doOpenModal(MODALS.CONFIRM, {
+                        title: __('Close Player'),
+                        subtitle: __('Are you sure you want to close the player and clear the current Queue?'),
+                        onConfirm: (closeModal) => {
+                          doSetPlayingUri({ uri: null });
+                          doClearQueueList();
+                          closeModal();
+                        },
+                      });
+                    } else {
+                      doSetPlayingUri({ uri: null });
+                    }
+                  }}
+                  icon={ICONS.REMOVE}
+                  button="primary"
+                  className="content__floating-close"
+                />
+              )}
+
+              {isReadyToPlay ? (
+                <FileRender className={classnames({ draggable: !isMobile })} uri={uri} />
+              ) : !collectionId && !canViewFile ? (
+                <div className="content__loading">
+                  <AutoplayCountdown
+                    nextRecommendedUri={nextListUri}
+                    doNavigate={() => setDoNavigate(true)}
+                    doReplay={() => doUriInitiatePlay({ uri, collectionId }, false, isFloating)}
+                    doPrevious={() => {
+                      setPlayNext(false);
+                      setDoNavigate(true);
+                    }}
+                    onCanceled={() => setCountdownCanceled(true)}
+                    skipPaid
+                  />
+                </div>
+              ) : (
+                <LoadingScreen status={__('Loading')} />
+              )}
+
+              {isFloating && (
+                <div className={classnames('content__info', { draggable: !isMobile })}>
+                  <div className="content-info__text">
+                    <div className="claim-preview__title" title={title || uri}>
+                      <Button
+                        label={title || uri}
+                        navigate={navigateUrl}
+                        button="link"
+                        className="content__floating-link"
+                      />
+                    </div>
+
+                    <UriIndicator link uri={uri} />
+                  </div>
+
+                  {playingCollection && collectionSidebarId !== collectionId && (
+                    <PlaylistCard
+                      id={collectionId}
+                      uri={uri}
+                      bodyOpen={playlistBodyOpen}
+                      enableCardBody={() => setPlaylistBody(!playlistBodyOpen)}
+                      customTitle={__('Now playing: --[Which Playlist is currently playing]--') + ' '}
+                      playItemsOnClick
```

Comment:
`playItemsOnClick`

Do you anticipate a future `PlaylistCard` mounted somewhere else without `playItemsOnClick`?  If not, maybe we can remove this prop from `PlaylistCard`, and just enable `playItemsOnClick` on the ClaimList inside it?